ReactWindows/ReactNative/ReactNative.csproj

```diff
@@ -113,7 +113,6 @@
     <Compile Include="Bridge\ReactInstance.cs" />
     <Compile Include="Bridge\DispatcherHelpers.cs" />
     <Compile Include="Bridge\JavaScriptBundleLoader.cs" />
-    <Compile Include="Bridge\NativeModuleBase.cs" />
     <Compile Include="Bridge\NativeModuleRegistry.cs" />
```

Comment:
Agreed, it's a big bite.  The folder change kind of needed a small example to justify it (and really test the feasibility of) it.  It helped us see our path going forward, but yes it bloats this PR.  

`NativeModuleBase` seems to be one of the major fundamental lynchpins.  We're able to to finally move modules with this working.